.github/workflows/artifacts.yml

```diff
@@ -0,0 +1,77 @@
+name: Build artifacts
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  alpine_build:
+    name: Alpine Build
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker-compose run build
+    - run: docker-compose run Profiler.Alpine
+    - run: docker-compose run package.alpine
+    - run: mkdir artifacts
+    - run: cp -a deploy/linux/. artifacts/
+    - uses: actions/upload-artifact@v2
+      with:
+        name: artifacts
+        path: artifacts
+
+  linux_build:
+    name: Linux Build
+    runs-on: ubuntu-16.04
```

Comment:
any reason why not something newer? e.g. `ubuntu-20.04` ? 
`ubuntu-16.04` has EOL this month 😄

Reply:
Oh, I was using the same what we had in signalfx, I will bump that.

Reply:
let's move to the future :)